bedrock/thunderbird/templates/thunderbird/index.html

```diff
@@ -15,9 +15,7 @@
   <h2>{{ high_res_img('thunderbird/landing/wordmark.png', {'alt': _('Thunderbird'), 'width': '304', 'height': '84'}) }}</h2>
 {% endblock %}
 
-{% block site_css %}{% endblock %}
-
-{% block extrahead %}
+{% block page_css %}
```

Comment:
As long as the download button is still visible and works, then I agree it's fine to change this one :smile: